### PR TITLE
Jorge

### PR DESCRIPTION
**Pull request by JorgeVidal29**
**Description**
Redesign of the login site and starter code to connect to database.
Passes flow and my testing

### CODE REVIEW

File: fortuna/src/LoginPopup.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+// Informational Login Popup component.
+class LoginPopup extends React.Component<{||}> {
+  render(): React.Node {
+    return (
+      <Popup trigger={<button className="button">What is Fortuna?</button>} modal>
+        {close => (
+	  <div className="modal">
+            <div className="header">What is Fortuna?</div>
+	    <div className="content">
```
Comment (SecondThread) [inline fortuna/src/LoginPopup.js:14]: Fix whitespace/indentation inconsistencies here please

File: fortuna/src/login/Login.js [hunk 2]
```diff
@@ -3,37 +3,120 @@
...
+const PORT = 3000;
 // Login component.
-class Login extends React.Component<{||}> {
+type Props = {||}; 
+type State = {|
+    response: string,
+    userName: string,
+    password: string,
+    responseToPost: string,
+  |};
```
Comment (SecondThread) [inline fortuna/src/login/Login.js:16]: nit: make the closing |} indentation match the indentation level of type (like you would with a for loop or method).

Thanks for fixing this though :)

File: fortuna/src/login/Login.js [hunk 3]
```diff
@@ -3,37 +3,120 @@
...
+//DO NOT USE THIS AS PART OF A BIGGER OBJECT.
+//ONCE SIGNUP API IS DONE PUT || AT THE END
+type SignupResponse = {
+    express: string,
+    message: string
+  };
```
Comment (SecondThread) [inline fortuna/src/login/Login.js:22]: Same nit, but again, thanks for fixing :)

**Review by SecondThread — CHANGES_REQUESTED** [review]
Pretty much perfect, only requesting changes because we need to add package-lock.json to the .gitignore so we don't have these massive autogenerated changes.

As soon as you get rid of the package-lock changes, ping me and I'll stamp

**Review by SecondThread — APPROVED** [review]

**Comment by JorgeVidal29** [comment]
> git fetch origin

**Pull request merged**